Doc/whatsnew/3.12.rst

```diff
@@ -159,7 +159,7 @@ Deprecated
   and tailor them to your needs.
   (Contributed by Erlend E. Aasland in :gh:`90016`.)
 
-* The exception representation \(type\[, value\[, traceback\]\]\) is now deprecated. (:gh:`89874`.)
+* The throw\(type, value, traceback\) signature is now deprecated. (:gh:`89874`.)
```

Comment:
The 3-arg signatures ``(type, value, traceback)`` of :meth:`~coroutine.throw` and :meth:`~generator.throw` are deprecated and may be removed in a future version of Python. Use the single-arg versions of these functions instead.